src/modules/72-triggers/pages/triggers/TriggerLandingPage/TriggerActivityHistoryPage/TriggerActivityHistoryPage.tsx

```diff
@@ -23,7 +23,7 @@ function TriggerActivityHistoryPage(): React.ReactElement {
   const { getString } = useStrings()
   const { orgIdentifier, projectIdentifier, pipelineIdentifier, accountId, triggerIdentifier } = useParams<
     PipelineType<
-      PipelinePathProps & {
+      TriggerPathProps & {
```

Comment:
```suggestion
      TriggerPathProps
```
TriggerPathProps has `triggerIdentifier`

```
export interface TriggerPathProps extends PipelinePathProps {
  triggerIdentifier: string
  triggerType?: string
  sourceRepo?: string
  artifactType?: string
  manifestType?: string
  scheduleType?: string
}
```